pkg/test/framework/components/opentelemetry/kube.go

```diff
@@ -31,7 +32,86 @@ type otel struct {
 }
 
 const (
-	appName = "opentelemetry-collector"
+	appName         = "opentelemetry-collector"
+	remoteOtelEntry = `
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: otel-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 55678
+      name: http-tracing-span
+      protocol: HTTP
+    hosts:
+    - "opentelemetry-collector.{INGRESS_DOMAIN}"
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: opentelemetry-collector
+  namespace: istio-system
+spec:
+  hosts:
+  - "opentelemetry-collector.{INGRESS_DOMAIN}"
+  gateways:
+  - otel-gateway
+  http:
+  - match:
+    - port: 55678
+    route:
+    - destination:
+        host: opentelemetry-collector
+        port:
+          number: 55678
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: opentelemetry-collector
+  namespace: istio-system
+spec:
+  host: opentelemetry-collector
+  trafficPolicy:
+    tls:
+      mode: DISABLE
+---`
+
+	extServiceEntry = `
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: opentelemetry-collector
+spec:
+  hosts:
+  # must be of form name.namespace.global
+  - opentelemetry-collector.istio-system.global
+  # Treat remote cluster services as part of the service mesh
+  # as all clusters in the service mesh share the same root of trust.
+  location: MESH_INTERNAL
+  ports:
+  - name: http-tracing-span
+    number: 55678
+    protocol: http
+  resolution: DNS
+  addresses:
+  # the IP address to which opentelemetry-collector.istio-system.global will resolve to
+  # must be unique for each remote service, within a given cluster.
+  # This address need not be routable. Traffic for this IP will be captured
+  # by the sidecar and routed appropriately.
+  - 240.0.0.2
```

Comment:
curious: should this be a non-routable address (ie: 172.16.0.1 or similar)?  how did we select this?

Reply:
I picked the default one from this example: https://istio.io/latest/docs/setup/install/multicluster/gateways/#configure-the-example-services